README.md

```diff
@@ -1,43 +1,66 @@
 # Lua on Unikraft
 
-This application prints "hello world from initrd" using Lua.
+## KraftKit
 
-To configure, build and run this application you need to have [kraft](https://github.com/unikraft/kraft) installed.
-
-Configure the application:
-```
-kraft configure
+```console
+kraft build -t lua-qemu-x86_64-9pfs -j $(nproc)
+kraft build -t lua-qemu-x86_64-initrd -j $(nproc)
+kraft build -t lua-qemu-arm64-9pfs -j $(nproc)
+kraft build -t lua-qemu-arm64-initrd -j $(nproc)
+kraft build -t lua-fc-x86_64-initrd -j $(nproc)
+kraft build -t lua-fc-arm64-initrd -j $(nproc)
 ```
 
-Build the application:
-```
-kraft build
-```
+## Build
 
-And finally, run the application:
-```
-kraft run -i helloworld.lua
+```console
+make distclean
+UK_DEFCONFIG=$(pws)/.config... make defconfig
```

Comment:
I think here we should give an example for the .... Otherwise, I would be tempted to run the command like it is.